src/python/turicreate/toolkits/drawing_classifier/_tf_drawing_classifier.py

```diff
@@ -106,35 +99,29 @@ def __init__(self, # validation_set, # net_params,
-        # TODO: Until the nn_spec in C++ isn't ready, ignore training with 
-        #       MXNet. When the nn_spec is ready, uncomment and edit the 
-        #       following lines to use the weights coming from C++
-        #       
-        ########################################################################
-        # for key in layers:
-        #     if 'bias' in key:
-        #         self.sess.run(_tf.assign(_tf.get_default_graph().get_tensor_by_name(key+":0"),
-        #             net_params[key].data().asnumpy()))
-        #     else:
-        #         if 'dense' in key:
-        #             if 'drawing_dense0_weight' in key:
-        #                 '''
-        #                 To make output of MXNET pool3 (NCHW) compatible with TF (NHWC).
-        #                 Decompose FC weights to NCHW. Transpose to NHWC. Reshape back to FC.
-        #                 '''
-        #                 mxnet_128_576 = net_params[key].data().asnumpy()
-        #                 mxnet_128_576 = _np.reshape(mxnet_128_576, (128, 64, 3, 3))
-        #                 mxnet_128_576 = mxnet_128_576.transpose(0, 2, 3, 1)
-        #                 mxnet_128_576 = _np.reshape(mxnet_128_576, (128, 576))
-        #                 self.sess.run(_tf.assign(_tf.get_default_graph().get_tensor_by_name(key+":0"),
-        #                                 mxnet_128_576.transpose(1,0)))
-        #             else:
-        #                 self.sess.run(_tf.assign(_tf.get_default_graph().get_tensor_by_name(key+":0"),
-        #                                 net_params[key].data().asnumpy().transpose(1, 0)))
-        #         else:
-        #             self.sess.run(_tf.assign(_tf.get_default_graph().get_tensor_by_name(key+":0"),
-        #                                 net_params[key].data().asnumpy().transpose(2, 3, 1, 0)))
+        for key in layers:
+            if 'bias' in key:
+                self.sess.run(_tf.assign(_tf.get_default_graph().get_tensor_by_name(key+":0"),
+                    net_params[key].data().asnumpy()))
+            else:
+                if 'dense' in key:
+                    if 'drawing_dense0_weight' in key:
+                        '''
+                        To make output of MXNET pool3 (NCHW) compatible with TF (NHWC).
+                        Decompose FC weights to NCHW. Transpose to NHWC. Reshape back to FC.
+                        '''
+                        mxnet_128_576 = net_params[key].data().asnumpy()
+                        mxnet_128_576 = _np.reshape(mxnet_128_576, (128, 64, 3, 3))
+                        mxnet_128_576 = mxnet_128_576.transpose(0, 2, 3, 1)
+                        mxnet_128_576 = _np.reshape(mxnet_128_576, (128, 576))
+                        self.sess.run(_tf.assign(_tf.get_default_graph().get_tensor_by_name(key+":0"),
+                                        mxnet_128_576.transpose(1,0)))
```

Comment:
Can the low-level conversions be factored out into a shared utility function? @shreyajain17 I thought we had some utilities for this kind of thing, after previous toolkits?

Reply:
We have some utility functions in `_tf_utils.py`. Basically I have it for all of the layers activity classifier used. But it would be extremely useful if @shantanuchhabra you could look for the layers you are converting in the utility file and add whatever conversions are not there for future.

Reply:
@shreyajain17 @nickjong Could we have those changes come in a later PR? For this PR, my goal was to enable the TF code that we had disabled earlier.

Reply:
Okay, but in general please strive to leave code in a better state whenever you touch it...

The corollary here is that it's generally bad form to comment out than comment back in large sections of code. You are now responsible for it, per `git blame` / `git annotate`

Reply:
Fair point, noted, thanks.